tools.templates/org.eclipse.tools.templates.freemarker/META-INF/MANIFEST.MF

```diff
@@ -2,12 +2,12 @@ Manifest-Version: 1.0
 Automatic-Module-Name: org.eclipse.tools.templates.freemarker
 Bundle-ManifestVersion: 2
 Bundle-SymbolicName: org.eclipse.tools.templates.freemarker
-Bundle-Version: 1.3.0.qualifier
+Bundle-Version: 1.3.100.qualifier
 Bundle-Name: %pluginName
 Bundle-Vendor: %providerName
 Require-Bundle: org.eclipse.core.runtime,
  org.eclipse.core.resources,
- org.freemarker;visibility:=reexport,
+ org.freemarker.freemarker;visibility:=reexport,
```

Comment:
@jonahgraham Seems we can either add a filter to suppress the API error here (as both bundles contain the same packages so the API isn't actually broken) or we remove the re-export and bump the major version and make this a breaking API change.

Maybe a point for discussion on the call, do we want to make a breaking API change in the next release?

Reply:
Decision in the [meeting](https://hackmd.io/@jonahgraham/cdt-call-september-13-2023) was to update freemarker to be package imports and remove the require bundle + reexport. That may lead to a some rare breakages for anyone that was relying on the reexport, but otherwise updating freemarker bundle is identified as API breakage anyway. This needs a N&N/API entry or similar.